### ISSUE TITLE
negative acknowledgement, but no packet pending

### ISSUE THREAD

**Issue by hyperion009** [riscv/riscv-openocd#975]
Hi
I am trying to set up the scenario: riscv(RTL)<--->Openocd <---> gdb. 
Here is my gdb command (target remote local:3333 in test.gdb) :
riscv64-unknown-elf-gdb  -x test.dgb hello.elf
![1701701569543](https://github.com/riscv/riscv-openocd/assets/30075118/93d63f35-71fa-451e-886e-9acdacefddfa)
Openocd has also reported:
![1701701677920](https://github.com/riscv/riscv-openocd/assets/30075118/2a829ddc-03fc-437c-beba-de610d5f7d1d)
Any comments are appreciated, thanks

**Comment by TommyMurphyTM1234** [comment]
More info needed - for example:

* What is your RTL target and how have you verified that it has been implemented (in FPGA or simulation?) correctly?
* Where did you obtain or how did you build OpenOCD?
* How are you invoking OpenOCD?
* What OpenOCD script(s) are you using?
* You probably also need to capture, compress and attach (don't copy/paste/embed it as it will be too long) a verbose `openocd -d3` log.

**Comment by hyperion009** [comment]
@TommyMurphyTM1234 thanks Tommy for your help. Here is my supplement:
What is your RTL target and how have you verified that it has been implemented (in FPGA or simulation?) correctly?

- It's an in-house RISC-V implementation. and I am doing a simulation on it. 

Where did you obtain or how did you build OpenOCD?

- Git clone https://github.com/riscv/riscv-openocd directly
- using the following comand to compile:

> ./bootstrap
> ./configure --prefix=/tools/openocd/ --enable-remote-bitbang
> make
> make install


How are you invoking OpenOCD?

- I invoked simualtion, openocd and gdb seperately in each of shell.
- Openocd has a successful dm initialization and entry the halt status.  
- failed at executing the first gdb debug command

What OpenOCD script(s) are you using?

- openocd.cfg in the attached zip file

You probably also need to capture, compress and attach (don't copy/paste/embed it as it will be too long) a verbose openocd -d3 log.

- opeocd.log in the attached zip file

zip file: 
[Uploading openocd.zip…]()
